docs/_articles/en/lwc/linting.md

```diff
@@ -0,0 +1,80 @@
+---
+title: Linting 
+lang: en
+---
+
+Linting provides errors about malformed code while you edit. VS Code enforces Salesforce’s ESLint rules. 
+
+## Prerequisites
+
+[The active LTS release](https://nodejs.org/en/about/releases/) of Node.js built with SSL support. 
+
+## Installation
+
+**For a New Project**
+
+If you've recently created a new SFDX project using the `force:project:create` command, your project contains a `package.json` file with the ESLint plugin already included. Make sure to run `npm install` in your project directory to install the dependencies from `package.json`. To start linting, run `npm run lint:lwc`.
```

Comment:
I tried these steps from scratch and on a brand new project the `lint:lwc` command will throw an error because there aren't any LWC components created yet for it to lint. Is it worth calling out that you need to create LWC components before linting? Maybe not, just thinking about it.